core/tests/protractor/subscriptions.js

```diff
@@ -50,13 +53,9 @@ describe('Subscriptions functionality', function() {
     browser.get('/profile/creator2subscriptions');
     browser.waitForAngular();
     element(by.css('.protractor-test-subscription-button')).click();
-    browser.get(general.USER_PREFERENCES_URL);
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'creator...');
-    expect(element.all(by.css(
-      '.protractor-test-subscription-name')).last().getText()).toMatch(
-      'creator...');
+    preferencesPage.get();
+    preferencesPage.expectFirstSubscriberToBe('creator...');
```

Comment:
I know it was "creator..." before, but let's match the username exactly instead. Otherwise the test is weak.

Reply:
done!!